scalafmt-core/shared/src/main/scala/org/scalafmt/config/Align.scala

```diff
@@ -61,6 +61,7 @@ import metaconfig.generic.Surface
   *   of the first line, as usual.
   */
 case class Align(
+    multiline: Boolean = false,
```

Comment:
Test cases looks great. Does this have to be a separate option? May be just apply it for `align!=none`?

Reply:
if we enable this code by default, everyone's code will change and people will complain. this is a potentially massive change.

Reply:
Yes, it's a valid point. But what about true by default?

Reply:
i'd leave it to you and @olafurpg to decide if you want to change defaults :)

i think it would be ok to include it as default in `align.preset = most` as it is advertised in the configuration as likely to be aggressive and also change in the future.

Reply:
True for `most` preset is optimal solution

Reply:
done. also added documentation.

Reply:
True for most sounds good.